java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -325,8 +325,8 @@ CompletableFuture<Long> linsert(
     CompletableFuture<String[]> brpop(String[] keys, double timeout);
 
     /**
-     * Inserts specified values at the tail of the <code>list</code>, only if <code>key</code> already
-     * exists and holds a list.
+     * Inserts all the specified values at the tail of the list stored at <code>key</code>, only if
+     * the list already exists.
```

Comment:
Split into two sentences. 
```
     * If <code>key</code> is a list, inserts all the specified values at the tail of the list. If
     * <code>key</code> is not a list, this performs no operation. 
```

Reply:
I feel like we should explicitly specify that this performs no operation if the key is not a list.

Reply:
updated